samples/XCT.Sample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -16,7 +16,7 @@
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
-    <TargetFrameworkVersion>v10.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v11.0</TargetFrameworkVersion>
```

Comment:
can you revert this change? This is causing the build error on CI.

Reply:
Is it possible to make CI support Android 11?

Reply:
Not sure about it.

Reply:
For now, let's revert it. Upgrading an API needs more work to guarantee that everything works as expected and changing deprecated APIs